### PR TITLE
Pigeonhole Sort using Kotlin

### LINKED ISSUE #2149: Pigeonhole Sort
- [x] C
- [x] C++
- [x] C# - @Harikrishnan6336
- [ ] CoffeeScript
- [ ] Dart - @heisastark
- [x] Go - @m-code12
- [x] Java
- [ ] JS - @sheetalssr
- [ ] Kotlin - @Anushka118
- [ ] Php
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by Anushka118**
Fixes #2149 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- implementing pigeonhole sort using Kotlin 

#### Languages Used:
- Kotlin 

#### Files Added:
- Pigeonhole_Sort.kt
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Anushka118** [comment]
@jainaman224 please have look at the PR.

File: Pigeonhole_Sort/Pigeonhole_Sort.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import java.util.*
+
+fun main(args: Array<String>) {
+
+    var n: Int = Integer.valueOf(readLine())    // taking input for size of the array
+    var arr = arrayOfNulls<Int>(n)              // declaring arr of size n
+    var read= Scanner(System.`in`)
```
Comment (atarax665) [inline Pigeonhole_Sort/Pigeonhole_Sort.kt:7]: space between read and =

Comment (Anushka118) [reply Pigeonhole_Sort/Pigeonhole_Sort.kt]: done

Comment (Anushka118) [reply Pigeonhole_Sort/Pigeonhole_Sort.kt]: please have a look at the file. @atarax665 @somya-kapoor

**Comment by Anushka118** [comment]
when would my pr be merged? @atarax665 @somya-kapoor @jainaman224 
it's been 4 days since my pr was approved.

**Pull request merged**